internal/apps/go.mod

```diff
@@ -8,10 +8,10 @@ require (
 )
 
 require (
-	github.com/DataDog/appsec-internal-go v1.4.0 // indirect
-	github.com/DataDog/go-libddwaf/v2 v2.2.3 // indirect
+	github.com/DataDog/appsec-internal-go v1.4.1 // indirect
```

Comment:
Why was this change made as part of this PR? Was it needed? I'm asking because I need to decide whether to backport this change into my v1.61.0 release branch to get the CI checks to pass.

Reply:
Update: I made a mistake while looking at this. I thought it was the main `go.mod` file, not the `internal/apps/go.mod` file. So this change doesn't matter, and I didn't need to revert it in https://github.com/DataDog/dd-trace-go/pull/2567.

It's all good, but sorry for the noise.